File: EQIS_env-main/react_sample/react/src/interfaces/notice/pwiImtrB.ts
import { baseInit, Ibase } from '../common/base';

export interface IpwiImtrB extends Ibase {
    pwiImtrNo: number;
    titlNm: string;
    pwiImtrSbc: string;
    pwiYn: string;
    supiFxgYn: string;
    popuYn: string;
    popuStrDtm: string;
    popuFnhDtm: string;
    useYn: string;
}

export const pwiImtrBInit = () => {
    const pwiImtrB: IpwiImtrB = {
        pwiImtrNo: 0,
        titlNm: '',
        pwiImtrSbc: '',
        pwiYn: '',
        supiFxgYn: '',
        popuYn: '',
        popuStrDtm: '',
        popuFnhDtm: '',
        useYn: '',
        ...baseInit(),
    };

    return pwiImtrB;
};

File: EQIS_env-main/react_sample/springboot/Spring/src/main/java/com/sample/basic/sample/notice/model/PwiImtrB.java
package com.sample.basic.sample.notice.model;

import com.sample.basic.cmm.model.Base;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * <pre>
 * 언어공통코드 상세
 * </pre>
 * 
 * @author user
 *
 */

@Data
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder
public class PwiImtrB extends Base{
	private int pwiImtrNo;
	private String titlNm;
	private String pwiImtrSbc;
	private String pwiYn;
	private String supiFxgYn;
	private String popuYn;
	private String popuStrDtm;
	private String popuFnhDtm;
	private String useYn;
}
